Clarification on product description (#171)
fiskaltrust/productdescription-de-doc: product-service-description/compliance-as-a-service/products/middleware.md

| Hardware                          | Mindestanforderung                                           |
| --------------------------------- | ------------------------------------------------------------ |
| Allgemeine Hardware Anforderungen | Ein Windows, Unix (_mit Mono Unterstützung_) oder Android basiertes System |

> | lokaler Speicher                  | ca. 500 MB (200 MB für die Middleware + 200 MB reserviert für Update Packages); optional: Speicher für SQLite DB (ca. 8-10Kb/Beleg) |

Reviewer: ```suggestion
| lokaler Speicher                  | ca. 500 MB (200 MB für die fiskaltrust.Middleware + 200 MB reserviert für Update Packages); optional: Speicher für SQLite DB (ca. 8-10Kb/Beleg) |
```

| Hardware Anschlüsse für TSE       | USB, SD, Micro-SD oder COM Port bei lokal angeschlossener TSE |

> | Internet Konnektivität            | zur Nutzung von Cloud TSE, Data-as-a-Service, Backup oder Konfigurationsupdate Services |

Reviewer: ```suggestion
| Internet Konnektivität            | Zur Nutzung von Cloud-TSE, Data-as-a-Service, Backup oder CashBox-Update über das fiskaltrust.Portal |
```

## Unterstützte Plattformen & TSEs

|                                                                                    | Windows                                 | Linux, macOS                                  | Android |
| ---------------------------------------------------------------------------------- | --------------------------------------- | --------------------------------------------- | ------- |

> | [A-Trust Cloud TSE](../features/basics/tse/cloud/a-trust.md)                       | ja                                      |                                               |         |

Reviewer: ```suggestion
| [A-Trust Cloud-TSE (a.sign TSE)](../features/basics/tse/cloud/a-trust.md)                       | ja                                      |                                               |         |
```

### Schnittstellendokumentation

> https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/communication

Reviewer: ```suggestion
https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc
```

### Datenspeicher

Folgende Optionen sind über die Konfiguration der Queue einstellbar:

|                                                                    | Windows | Linux, macOS | Android |
| ------------------------------------------------------------------ | ------- | ------------ | ------- |
| [SQLite-Storage](../features/supported-databases/sqlite.md)        | ja      | ja           | ja      |

> | [EF-Storage](../features/supported-databases/entity-framework.md)  | ja      | ja           |         |

Reviewer: ```suggestion
| [EF-Storage](../features/supported-databases/entity-framework.md)  | ja      | ja           |  nein        |
```
